cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go

```diff
@@ -81,6 +86,11 @@ func printPolicyCmd() *cobra.Command {
 
 func getDocumentName(cmd *cobra.Command) (bootstrap.PolicyName, error) {
 	val := bootstrap.PolicyName(cmd.Flags().Lookup("document").Value.String())
+
+	if val == "" {
+		return "", nil
```

Comment:
This should also probably be a validation error?

Reply:
I guess if no policy is passed, then we give empty line in console, we could actually make this intuitive and return a message saying "No policy doc provided" or something like that

Reply:
@Atharva-Shinde could you please take care of this ?

Reply:
Thanks @Ankitasw I will make changes today itself :)